### PR TITLE
Examples of Redux and GraphQL with Riot 

### PR DESCRIPTION
**Pull request by renanpalmeira**
I created two new examples in riot using Redux (tested in production in my company) and GraphQL (yet in beta/tests for me). But in this two PR the examples working.

Thanks

### CODE REVIEW

File: graphql/app/components/app.tag [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  </div>
+
+  <script>
+    import gql from 'graphql-tag';
+    
+    this.name = 'Before GraphQL'
+  
+    this.on('mount', () => {
+
+      this.refs.btn.onclick = () => {
```
Comment (GianlucaGuarini) [inline graphql/app/components/app.tag]: why don't you use the riot events here?

File: redux/app/components/app.tag [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<app>
+
+  <div class="app">
+    <h3>Riot with Redux</h3>
+
+    <p>
+        { this.store.getState().sampleReducer.name }
```
Comment (GianlucaGuarini) [inline redux/app/components/app.tag]: Probably this should be somewhere else and the app should only receive the object it needs as option.

File: redux/app/components/app.tag [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    <button ref="btn">
+      Initial fetch
+    </button>
+  </div>
+
+  <script>
+    this.on('mount', () => {
+
+      this.refs.btn.onclick = () => {
```
Comment (GianlucaGuarini) [inline redux/app/components/app.tag]: Please prefer riot events

File: redux/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader'
+      }
+    ]
+  },
+  devServer: {
+    disableHostCheck: true,   // That solved it
```
Comment (GianlucaGuarini) [inline redux/webpack.config.js]: what issue is it solving? Please provide more info

**Review by GianlucaGuarini — COMMENTED** [review]
Thanks, please fix the issues I have found and I will be happy to merge it

**Comment by renanpalmeira** [comment]
Hi @GianlucaGuarini, I tried fix problems found by you... I waiting your feedback

Thanks :smile:

**Comment by GianlucaGuarini** [comment]
thank you @RenanPalmeira

**Pull request merged**